profiles/views.py

```diff
@@ -127,3 +127,14 @@ class ViewedCompanyList(ListCreateAPIView):
     def get_queryset(self):
         user_id = self.request.user.id
         return ViewedCompany.objects.filter(user=user_id)
+
+
+class GetUserId(ListAPIView):
```

Comment:
There should be NO VERBS in view names.

Reply:
Thank you for feedback. I have deleted it at all. But I'll remember about verbs